types/react-native/index.d.ts

```diff
@@ -162,6 +164,31 @@ interface EmitterSubscription extends EventSubscription {
     remove(): void;
 }
 
+interface DEPRECATED_RCTExport<T> {
+	getConstants(): Function;
+}
+
+interface TurboModule extends DEPRECATED_RCTExport<void> {}
+
+interface Spec extends TurboModule {
+    addListener(eventName: string): void;
+    removeListeners(count: number): void;
+}
+
+declare class TurboModuleRegistry {
+	get<TurboModule>(name: string): (legacyModule: any) => TurboModule
+}
+
+interface NativeTVNavigationEventEmitter {
+	addListener(eventName: string, callback: Function): NativeEventEmitter;
+}
+ 
+declare class TVEventHandler {
+	new (): TVEventHandler;
```

Comment:
This should be redundant.

Reply:
changes are added - 0ec12eb6d3319739d960f1238a5a1495807535fe